src/qibocal/protocols/randomized_benchmarking/utils.py

```diff
@@ -154,82 +264,43 @@ def data_uncertainties(data, method=None, data_median=None, homogeneous=True):
 
 class RB_Generator:
     """
-    This class generates random single qubit cliffords for randomized benchmarking.
+    This class generates random two qubit cliffords for randomized benchmarking.
     """
 
-    def __init__(self, seed):
+    def __init__(self, seed, file=None):
         self.seed = seed
         self.local_state = (
             np.random.default_rng(seed)
             if seed is None or isinstance(seed, int)
             else seed
         )
 
-    def random_index(self, gate_list):
-        """
-        Generates a random index within the range of the given gate list.
-
-        Parameters:
-        - gate_list (list): Dict of gates.
+        if file is not None:
+            self.two_qubit_cliffords = load_cliffords(file)
+            self.file = file
+        else:
+            self.file = None
```

Comment:
if file is not None self.file is equal to file, otherwise (so if it is None) self.file is None.
So this should work
```suggestion
        if file is not None:
            self.two_qubit_cliffords = load_cliffords(file)

        self.file = file
```